webextension/background/auth.js

```diff
@@ -9,7 +9,13 @@ window.auth = (function () {
   let authHeader = null;
   let sentryPublicDSN = null;
 
-  chrome.storage.local.get(["registrationInfo"], (result) => {
+  chrome.storage.local.get(["registrationInfo"], catcher.watchFunction((result) => {
+    if (chrome.runtime.lastError) {
```

Comment:
Hmm. Maybe this is because you're using `chrome` instead of `browser`?

<img width="320" alt="screen shot 2017-03-02 at 7 31 29 pm" src="https://cloud.githubusercontent.com/assets/96396/23537134/db4dfbe2-ff7e-11e6-8ae5-508ab68b90b6.png">